### PR TITLE
use auto-built foreman-ansible-modules docs

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -12,7 +12,7 @@ end
 
 desc 'Start server'
 task :server do
-  sh "jekyll serve --watch --incremental"
+  sh "jekyll serve --no-watch --incremental"
```
Comment (ekohl) [inline Rakefile]: What's the reason for this change?

Comment (evgeni) [reply Rakefile]: me commiting too much :)

Comment (evgeni) [reply Rakefile]: (watch would bomb out on my machine for some reason, blame inotify, so I disabled it)

**Review by ehelms — APPROVED** [review]
